index.html

```diff
@@ -143,8 +143,7 @@ <h3 class="center">Our work reflected in numbers</h3>
       </li>
       <li class="section center">
         <h4>Number of people helped out</h4>
-        <p>A lot
-          <3</p>
+        <p>A lot &lt</p>
```

Comment:
@robeerob Oh yeah, I somehow deleteted the 3
It should be `<p>A lot &lt;3</p>` instead.
And I feel like the "<3" fits more, because it includes the opening tag which is used throughout every html file.